systemtests/utils/utils.go

```diff
@@ -53,14 +54,14 @@ func StopOnError(testFailed bool) {
 }
 
 // ConfigCleanupCommon performs common cleanup after each test
-func ConfigCleanupCommon(t *testing.T, nodes []TestbedNode) {
+func ConfigCleanupCommon(t *testing.T, nodes []stu.TestbedNode) {
 
 	if !OkToCleanup(t.Failed()) {
 		return
 	}
 
 	for _, node := range nodes {
-		cmdStr := "sh -c 'sudo $GOSRC/github.com/contiv/netplugin/scripts/cleanup'"
```

Comment:
You know, I'm surprised that change is in there, must be an artifact of changes I did earlier that didn't get vendored in or something.

I can get rid of it if it is an issue.